packages/vue-prism-editor/src/Editor.ts

```diff
@@ -306,13 +307,11 @@ export const PrismEditor = Vue.extend({
       // console.log(navigator.platform);
       const { tabSize, insertSpaces, ignoreTabKey } = this;
 
-      if (this.$listeners.keydown) {
-        // onKeyDown(e);
-        this.$emit('keydown', e);
+      // onKeyDown(e);
+      this.$emit('keydown', e);
```

Comment:
`this.$listeners` has been removed in vue 3 (or is null as far as I can tell) which triggers an undefined error.

Reply:
After investigation, `this.$listeners` exists only if it has been implemented, with an `@click=` or an `@keydown=` 
It should be protected against null.
Since it is deprecated anyway, I removed it completely.

Reply:
yep they've merged `$listeners` to `$attrs`. So `$attrs` has both `$attrs` and `$listeners` in vue 3